app/PickMe_nebula0/app/src/main/java/com/example/pickme_nebula0/organizer/activities/OrganizerHomeActivity.java

```diff
@@ -17,49 +19,27 @@
 import com.google.android.material.tabs.TabLayout;
 import com.google.android.material.tabs.TabLayoutMediator;
 
-/**
- * Activity for the organizer's home screen.
- *
- * Displays a tabbed interface with two categories: "Past" and "Ongoing" events. Provides functionality
- * for navigating to the "Create Event" activity and toggling between event categories.
- *
- * @see OrganizerPastFragment
- * @see OrganizerOngoingFragment
- * @see OrganizerCreateEventActivity
- */
```

Comment:
I think you made a mistake resolving merge conflicts.
These java docs are important and shouldn't be deleted.
Please resolve, thanks